src/Backend/Core/Layout/Sass/screen.scss

```diff
@@ -1,8 +1,8 @@
 @charset 'UTF-8';
 
 // Font Awesome
-$fa-font-path: '../../../../../bower_components/fontawesome/fonts';
-@import '../../../../../bower_components/fontawesome/scss/font-awesome';
+$fa-font-path: '/fonts';
```

Comment:
Maybe we should wait with this for now, as it is not related to the PR, and create a separate PR for this.